packages/vkui/src/lib/floating/types/component.ts

```diff
@@ -59,4 +59,11 @@ export interface FloatingComponentProps
    * чтобы всплывающий элемент вместился в эту область видимости.
    */
   onPlacementChange?: OnPlacementChange;
+  /**
+   * При использовании свойства `hideWhenReferenceHidden` колбэк срабатывает, когда
+   * целевой элемент вышел за область видимости(hidden=true) и вернулся обратно(hidden=false).
+   *
+   * > При первом появлении всплывающего элемента колбэк будет вызван со значением `false`
+   */
```

Comment:
```suggestion
  /**
   * Событие скрытия / раскрытия компонента при использовании свойства `hideWhenReferenceHidden`.
   *
   * > Стоит иметь ввиду, что событие также будет вызвано и при новом рендере компонента
   */
```
будет ли `false` зависит от того, попадет ли компонент в область видимости, при использования `disableFlipMiddleware` будет вероятность того, что компонент будет за пределами